web/quickfeed_service.go

```diff
@@ -85,9 +85,23 @@ func (s *QuickFeedService) UpdateCourse(ctx context.Context, in *connect.Request
 	scmClient, err := s.getSCM(ctx, in.Msg.GetScmOrganizationName())
 	if err != nil {
 		s.logger.Errorf("UpdateCourse failed: could not create scm client for organization %s: %v", in.Msg.GetScmOrganizationName(), err)
-		return nil, connect.NewError(connect.CodeNotFound, err)
+		return nil, connect.NewError(connect.CodeNotFound, fmt.Errorf("unable to connect to the gitHub organization: %s", in.Msg.GetScmOrganizationName()))
```

Comment:
In the interest of consistency across all service methods, replace these return types with just this:
```go
return nil, connect.NewError(connect.CodeNotFound, errors.New("unable to connect to the GitHub organization for the course"))
```
The logs will have more information if this should ever happen, but right now, we don't have the course organization name available in `UpdateEnrollments`, `UpdateGroup`, and `DeleteGroup`. We could make this common error message a global variable that we can reuse instead of having slightly different error messages for each of these.
```go
var scmConnectErr = connect.NewError(connect.CodeNotFound, errors.New("unable to connect to the GitHub organization for the course"))

return nil, scmConnectErr
```